docs/testing-manual.md

```diff
@@ -9,6 +9,24 @@ geekdocFilePath: testing-old.md
 
 {{< toc >}}
 
+## Setup backend
+
+### ownCloud 10
+
+- set up the [ownCloud 10 backend]({{< ref "backend-oc10.md" >}})
+- clone and install the [testing app](http://github.com/owncloud/testing) into ownCloud
+
+### oCIS
+In order to run the acceptance tests you need to run oCIS using the oCIS storage driver. Also, you need to enable basic auth on the server with these environment variables.
+
+`PROXY_ENABLE_BASIC_AUTH=true STORAGE_HOME_DRIVER=owncloud STORAGE_USERS_DRIVER=owncloud`
+
+- set up the [oCIS backend]({{< ref "backend-ocis.md" >}})
+  - if you are a Mac user, you need to start the server with additional environment variables: `STORAGE_HOME_DATA_SERVER_URL='http://host.docker.internal:9155/data' STORAGE_DATAGATEWAY_PUBLIC_URL='https://host.docker.internal:9200/data' STORAGE_USERS_DATA_SERVER_URL='http://host.docker.internal:9158/data' STORAGE_FRONTEND_PUBLIC_URL='https://host.docker.internal:9200' PROXY_ENABLE_BASIC_AUTH=true PROXY_OIDC_ISSUER='https://host.docker.internal:9200' IDP_INSECURE='true' IDP_IDENTIFIER_REGISTRATION_CONF='<web-path>/tests/acceptance/mac-identifier-registration.yml' IDP_ISS='https://host.docker.internal:9200' IDP_TLS='true'` (`<web-path>` needs to be replaced with the path of your local clone of ownCloud Web)
+
+## Setup the owncloud test Middleware
+To run the tests, we need to run the [owncloud-test-middleware](https://github.com/owncloud/owncloud-test-middleware). You can find the instructions on how to run the middleware server from its [github page](https://github.com/owncloud/owncloud-test-middleware#starting-the-server)
+
 ## Setup Selenium
```

Comment:
`setup selenium` section is moved after the `server setup` & `middleware setup` as the selenium docker needs to run with the `filesForUpload` from the middleware code base.